jina/peapods/peas/__init__.py

```diff
@@ -363,8 +363,8 @@ def role(self) -> 'PeaRoleType':
 
     @property
     def _is_inner_pea(self) -> bool:
```

Comment:
We can remove this I think, this is only used in a test

Reply:
sure, removed and changed in the test (which is not working anyway of course atm)